### DISCUSSION TITLE
Are there any restrictions on the use of Deta?

### DISCUSSION THREAD

**Discussion by lete114** [deta/cloud-docs#300] [General] [answered]
![visitor badge](https://visitor_badge.deta.dev/?pageID=github.orgs.deta.discussions.300)

Since I checked other discussions earlier, I learned that Deta Base ([#280](https://github.com/orgs/deta/discussions/280)) is unlimited and did not find anything about usage restrictions, such as request limits

If there is a request limit, what is the maximum number of requests per day (or per month)

Since I wrote a stats badge, I'm considering whether Deta has a usage limit [visitor-badge](https://github.com/Lete114/visitor-badge/blob/main/README_EN.md)

There is also the problem of getting the requested ip address ([#282](https://github.com/orgs/deta/discussions/282))

**Comment by jnsougata** [comment, marked as answer]
There's no request limit as such! but max allowed TCP connection is 100

**Reply by lete114** [reply]
Does that mean only 100 requests per second can be processed?

**Reply by jnsougata** [reply]
Noope, you can do more than 100/s if the server supports it. For example it might be 1 TCP connection for all 100 requests totally depends on how a server is configured.

**Comment by hosenur** [comment]
I think there is probably a limit to the app size.

**Reply by zawsq** [reply]
250 mb
